pom.xml

```diff
@@ -1226,7 +1226,16 @@
             <encoding>UTF-8</encoding>
             <maxmem>1024m</maxmem>
             <fork>true</fork>
+            <compilerId>javac-with-errorprone</compilerId>
```

Comment:
move it into a profile and turned on the compiler warnings since the profile itself is optional.